Sources/BuildTool/BuildTool.swift

```diff
@@ -196,3 +199,424 @@ struct Lint: AsyncParsableCommand {
         return try String(data: data, encoding: .utf8)
     }
 }
+
+@available(macOS 14, *)
+struct SpecCoverage: AsyncParsableCommand {
+    static let configuration = CommandConfiguration(abstract: "Print information about which spec points are implemented")
+
+    enum Error: Swift.Error {
+        case unexpectedStatusCodeLoadingCommit(Int)
+        case unexpectedStatusCodeLoadingSpec(Int)
+        case conformanceToNonexistentSpecPoints(specPointIDs: [String])
+        case couldNotFindTestTarget
+        case malformedSpecOneOfTag
+        case specUntestedTagMissingComment
+    }
+
+    /**
+     * A representation of the chat features spec Textile file.
+     */
+    private struct SpecFile {
+        struct SpecPoint: Identifiable {
+            var id: String
+            var isTestable: Bool
+
+            init?(specLine: String) {
+                // example line that corresponds to a testable spec point:
+                // ** @(CHA-RS4b)@ @[Testable]@ Room status update events must contain the previous room status.
+                // (This `Testable` is a convention that’s being used only in the Chat spec)
+
+                let specPointLineRegex = /^\*+ @\((.*?)\)@( @\[Testable\]@ )?/
+
+                // swiftlint:disable:next force_try
+                guard let match = try! specPointLineRegex.firstMatch(in: specLine) else {
+                    return nil
+                }
+
+                id = String(match.output.1)
+                isTestable = match.output.2 == nil ? false : true
```

Comment:
Isn't it just `isTestable = match.output.2 != nil`?

Reply:
Ah, yes! Changed.